cli/ballerina-cli/src/main/resources/cli-help/ballerina-deprecate.help

```diff
@@ -25,5 +25,8 @@ OPTIONS
 
 EXAMPLES
 
```

Comment:
Shouldn't we update the `synopsis` section also?
```
bal deprecate [OPTIONS] <org>/<name>[:<version>]
```

Reply:
Addressed with 98399e6b026b8e4cd874b580071f4bef435dedbe